### PR TITLE
FAQ added.

### PR DESCRIPTION
**Pull request by panasevychol**
FAQ added.

### CODE REVIEW

**Comment by panasevychol** [comment]
That's how it looks like:
image

File: index.html [hunk 1]
```diff
@@ -70,6 +71,56 @@ <h2>Download for your platform</h2>
...
+    <div class="Description">
+      <div class="DescriptionInnerBlock">
+
+        <h2>FAQ</h2>
+        <br>
+
+        <div class="FAQ" id="FAQ">
+
+          <div>
+            <h3>Are Python and Go setups on macOS/Windows mandatory to run the AEA Manager?</h3>
```
Comment (5A11) [inline index.html:83]:
```suggestion
            <h3>Are Python and Go installations on macOS/Windows mandatory to run the AEA Manager?</h3>
```

File: index.html [hunk 2]
```diff
@@ -70,6 +71,56 @@ <h2>Download for your platform</h2>
...
+            <p>
+              Yes. Python is required as the AEA framework is currently in  Python and Go is required by most agents. If you skip any of the above, you will not be able to run the application.
+            </p>
+          </div>
+          <br>
+
+          <div>
+            <h3>I have closed or interrupted the setup of Python/Go/Microsoft Visual C++ Build Tools on the first run of the application. How can I fix that?</h3>
```
Comment (5A11) [inline index.html:91]:
```suggestion
            <h3>I have closed or interrupted the installation of Python/Go/Microsoft Visual C++ Build Tools on the first run of the application. How can I fix that?</h3>
```

File: index.html [hunk 3]
```diff
@@ -70,6 +71,56 @@ <h2>Download for your platform</h2>
...
+            <p>
+              Simply restart the AEA Manager application and it will ask you to install the missing dependencies.
+            </p>
+          </div>
+          <br>
+
+          <div>
+            <h3>Is it mandatory to install Microsoft Visual C++ Build Tools on Windows to run AEA Manager?</h3>
```
Comment (5A11) [inline index.html:99]:
```suggestion
            <h3>Is it mandatory to install Microsoft Visual C++ Build Tools on Windows to run the AEA Manager?</h3>
```

**Review by 5A11 — APPROVED** [review]

**Pull request merged**